lib/servers/iXonServer/qt4reactor.py

```diff
@@ -0,0 +1,253 @@
+# Copyright (c) 2001-2008 Twisted Matrix Laboratories.
```

Comment:
@jayich @gregllong, I just tried to remove this and for an unknown reason this slows the camera down to unusable